databricks/koalas/frame.py

```diff
@@ -484,10 +484,16 @@ def calculate_columns_axis(*cols):
         else:
             raise ValueError("No axis named %s for object type %s." % (axis, type(axis)))
 
+    def _kser_for(self, label):
```

Comment:
@ueshin, I manually debugged the code path being executed. Which code path is the concern for this shortcut? and, can we move this shortcut into `DataFrame.__getitem__` or `LocIndexer`?